cedar-policy/CHANGELOG.md

```diff
@@ -64,11 +68,13 @@ method checks the request against the schema provided and the
   rejected. E.g., `foo::principal` is an error and is not parsed as
   `principal`. Variables qualified by a namespace of any size comprised entirely
   of Cedar keywords are correctly rejected. E.g., `if::then::else::principal` is an error.
-  (#594 and #596)
+  (#594 and #597)
 - The entity type tested for by an `is` expression may be an identifier shared
   with a builtin variable. E.g., `... is principal` and `... is action` are now
-  accepted by the Cedar parser. (#558)
-- Policies containing the literal `i64::MIN` can now be properly converted to JSON ESTs (#601, resolving #596)
+  accepted by the Cedar parser. (#595, resolving #558)
+- Policies containing the literal `i64::MIN` can now be properly converted to JSON ESTs. (#601, resolving #596)
+- EST attributes can be special identifiers such as `principal`, `then`, and `true`. (#628, resolving #604)
```

Comment:
IIRC, an EST could use these attributes, it's just the AST to EST conversion failed when they were present.